code/modules/research/message_server.dm

```diff
@@ -78,12 +78,11 @@ GLOBAL_LIST_INIT(message_servers, list())
 	var/active = 1
 	var/decryptkey = "password"
 
-/obj/machinery/message_server/New()
+/obj/machinery/message_server/Initialize()
 	GLOB.message_servers += src
 	decryptkey = GenerateKey()
 	send_pda_message("System Administrator", "system", "This is an automated message. The messaging system is functioning correctly.")
-	..()
-	return

```

Comment:
This was running code before parent New() that would now happen after New().

Does this matter in this case?